lib/xml_builder.ex

```diff
@@ -339,6 +338,25 @@ defmodule XmlBuilder do
     ]
   end
 
+  defp format_children(list, level, options) when is_list(list) do
+    format_char = formatter(options).line_break()
```

Comment:
Can we use the naming `line_break = formatter(options).line_break()`

Because the line break might not necessarily be a single "char", could be an empty string `""` or line feed + new line `"\r\n`